### ISSUE TITLE
Does this project actually work?

### ISSUE THREAD

**Issue by mjschultz** [heroku-python/django-postgrespool#33]
Looking at the code, it seems like [`db_pool`](https://github.com/kennethreitz/django-postgrespool/blob/master/django_postgrespool/base.py#L28) is created at import to properly proxy to the sqlalchemy pool manager as expected.

However, as of the changeset in https://github.com/kennethreitz/django-postgrespool/pull/21 it seems like the `db_pool` is no longer used to connect to the database and it should fallback to the default django psycopg2 drivers.

The only reference to the `db_pool` object that should do the pooling magic is in the `_dispose` method. So it seems like this is a simple wrapper around the default Django backend.

Perhaps I am missing some DB-API knowledge or Django magic though?

**Comment by lcd1232** [comment]
@mjschultz yes it works. By the way this project haven't updated for a long time so I created fork with various fixes - https://github.com/lcd1232/django-postgrespool2

**Comment by mjschultz** [comment]
To be clear, this project does not work.

Your fork should work because it adds back the `get_new_connection` that uses the `db_pool`.

**Comment by edevil** [comment]
@kennethreitz So... this project does not work?

**Comment by kennethreitz** [comment]
@lcd1232 can you send a pull request?

**Comment by edevil** [comment]
@lcd1232 ping

**Comment by matthewh** [comment]
This project does not work - use the fork at https://github.com/lcd1232/django-postgrespool2.

**Comment by toanant** [comment]
@lcd1232 any plan to send PR with fork's changes to this repo?

**Comment by vkolova** [comment]
Neither this or the fork work for me. (Using Django 2)
